src/git.js

```diff
@@ -287,7 +287,7 @@
     Git.prototype.listRemote = function (args, then) {
         if (!then && typeof args === "function") {
             then = args;
-            args = [];
+            args = undefined;
```

Comment:
By setting this to `undefined` the array passed to `_run` on line [297](https://github.com/seancolyer/git-js/blob/master/src/git.js#L297) will have `undefined` added to the end of it, which is why this is defaulted to an empty array.